framework/src/modules/keys/utils/verify.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Account } from '@liskhq/lisk-chain';
+import { verifyData } from '@liskhq/lisk-cryptography';
+import { AccountKeys } from '../types';
+
+export const isMultisignatureAccount = (account: Account<AccountKeys>): boolean =>
+	!!(
+		(account.keys.mandatoryKeys.length > 0 || account.keys.optionalKeys.length > 0) &&
+		account.keys.numberOfSignatures
+	);
+
+export const validateSignature = (
+	publicKey: Buffer,
+	signature: Buffer,
+	bytes: Buffer,
```

Comment:
transactionBytes?

Reply:
This was originally like this; renamed as no opinion on this.